src/mono/wasm/runtime/assets.ts

```diff
@@ -38,6 +38,19 @@ const skipBufferByAssetTypes: {
     "dotnetwasm": true,
 };
 
+const skipInSnapshotByAssetTypes: {
+    [k: string]: boolean
+} = {
+    "resource": true,
+    "assembly": true,
+    "pdb": true,
```

Comment:
are memory snapshots enabled when debugging? should they be?

Reply:
Yes, they are.